From b6206f0f6ee6de5071bcc7c8aafad2144ca3ee78 Mon Sep 17 00:00:00 2001
From: Wilson Yang <wilson_yang@codeaurora.org>
Date: Wed, 27 Nov 2013 15:21:43 -0800
Subject: [PATCH 17/18] wlan: Non-sec wd bark due to  excessive logging

In wlan_hdd_tx_rx.c: function hdd_start_xmit will
cause excessive logs during TDLS transimmission
when there are chnanges in link state

Change-Id: Ib7a4defee50ab68bf3a5199541a5843213bf7ce1
CRs-fixed: 571471

diff --git a/CORE/HDD/src/wlan_hdd_tx_rx.c b/CORE/HDD/src/wlan_hdd_tx_rx.c
index 9a874ee..438dc26 100644
--- a/CORE/HDD/src/wlan_hdd_tx_rx.c
+++ b/CORE/HDD/src/wlan_hdd_tx_rx.c
@@ -628,7 +628,7 @@ int hdd_hard_start_xmit(struct sk_buff *skb, struct net_device *dev)
    {
       if (pAdapter->wmm_tx_queue[ac].count >= HDD_TX_QUEUE_LOW_WATER_MARK)
       {
-          VOS_TRACE( VOS_MODULE_ID_HDD, VOS_TRACE_LEVEL_WARN,
+          VOS_TRACE( VOS_MODULE_ID_HDD, VOS_TRACE_LEVEL_INFO,
                      "%s: Best Effort AC Tx queue is 3/4th full", __func__);
           pAdapter->isVosLowResource = VOS_TRUE;
       }
-- 
1.8.2.1

